website/docs/components/side-nav/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,126 @@
+## Usage
+
+### When to use
+
+- To display the primary navigation of the application on a page.
+
+### When not to use
+
+- To move between views within the same context or page, consider [Tabs](#).
+
+## Header
+
+The navigation header displays persistent UI elements that give users quick access to important sections and resources within the application or platform.
+
+### Home link
+
+The home link gives users consistent and quick access to the application's main page or section. Generally, the home or dashboard.
+
+For cloud products, the icon set should always be the HCP service icon. For standalone or open-source products, it should be the product’s service icon (e.g. terraform).
+
+![Home link in side-nav's header](/assets/components/side-nav/header-logo.png)
+
+### Help dropdown
+
+Use the help dropdown to provide users with access to support and helpful resources that can be easily accessible from anywhere within the application—for example, links to the help center, documentation, or tutorials.
+
+Avoid placing non-help related links or actions, such as user settings or navigation links.
+
+![Help dropdown menu in side-nav's header](/assets/components/side-nav/help-dropdown.png)
+
+### User dropdown
+
+Use the user dropdown to give users quick and easy access to their settings and preferences. The menu should contain links or actions related to the user's profile, settings, and/or preferences. 
+
+Avoid placing links to unrelated pages or actions, such as support or navigation items.
+
+![User dropdown menu in side-nav's header](/assets/components/side-nav/user-dropdown.png)
+
+## Body
+
+The body consists of a group of sections with vertical lists of links to the most important parts of the application.
```

Comment:
```suggestion
The body consists of a group of sections with vertical lists of links, typically to the most important parts of the application.
```

This would provide for incorporating @didoo's feedback without too much churn, I think.

Reply:
Thanks, Melanie. I took your suggestion and rephrased the paragraph a bit, hoping it's clearer now.